resources/js/components/user-groups/PublishForm.vue

```diff
@@ -1,107 +1,91 @@
-                        </div>
-                    </div>
-                    <publish-field-meta
-                        :config="{ handle: 'roles', type: 'user_roles' }"
-                        :initial-value="roles">
-                        <div slot-scope="{ meta, value, loading }">
-                            <relationship-fieldtype
-                                v-if="!loading"
-                                :config="{ handle: 'roles', type: 'user_roles', mode: 'select' }"
-                                :value="value"
-                                :meta="meta"
-                                handle="roles"
-                                @input="roles = $event" />
-                        </div>
-                    </publish-field-meta>
-                    <small class="help-block text-red mt-1 mb-0" v-if="errors.roles" v-text="errors.roles[0]" />
-                </div>
-
-            </publish-fields-container>
-            <div class="py-2 border-t flex justify-between">
-                <a :href="action" class="btn" v-text="__('Cancel') "/>
-                <button type="submit" class="btn-primary" @click="save">{{ __('Save') }}</button>
+    <div>
+
+        <header class="mb-3">
+            <breadcrumb :url="cp_url('user-groups')" :title="__('User Groups')" />
+            <div class="flex items-center">
+                <h1 class="flex-1" v-text="title" />
+                    <dropdown-list class="mr-2" v-if="canEditBlueprint">
+                        <dropdown-item :text="__('Edit Blueprint')" :redirect="actions.editBlueprint" />
+                    </dropdown-list>
+
+                    <button
+                        class="btn-primary"
+                        @click.prevent="save"
+                        v-text="__('Save')" />
+
+                <slot name="action-buttons-right" />
             </div>
-        </div>
+        </header>
+
+        <publish-container
+            v-if="fieldset"
+            ref="container"
+            :name="publishContainer"
+            :blueprint="fieldset"
+            :values="values"
+            :reference="initialReference"
+            :meta="meta"
+            :errors="errors"
+            @updated="values = $event"
+        >
+            <div slot-scope="{ container, setFieldValue, setFieldMeta }">
+                <publish-sections
+                    :enable-sidebar="false"
+                    :can-toggle-labels="true"
+                    @updated="setFieldValue"
+                    @meta-updated="setFieldMeta"
+                    @focus="container.$emit('focus', $event)"
+                    @blur="container.$emit('blur', $event)"
+                ></publish-sections>
+            </div>
+        </publish-container>
+
+    </div>
 </template>
 
 
 <script>
+import HasHiddenFields from '../data-list/HasHiddenFields';
+
 export default {
 
+    mixins: [
+        HasHiddenFields,
+    ],
+
+    components: {
+        ChangePassword,
+    },
```

Comment:
This needs to either be imported (`import ChangePassword from '../users/ChangePassword.vue'`) or just removed.

I'm guessing you copied the user form and missed removing this, so just remove it. 😄